Ginger/GingerCoreNET/Run/GingerRunner.cs

```diff
@@ -2870,122 +2889,140 @@ public void RunActivity(Activity activity, bool doContinueRun = false, bool stan
 
             try
             {
-                //check if Activity is allowed to run
-                if (CurrentBusinessFlow == null ||
-                    activity.Acts.Count == 0 || //no Actions to run
-                        activity.GetType() == typeof(ErrorHandler) ||//don't run error handler from RunActivity
-                            activity.CheckIfVaribalesDependenciesAllowsToRun(CurrentBusinessFlow, true) == false || //Variables-Dependencies not allowing to run
-                                (FilterExecutionByTags == true && CheckIfActivityTagsMatch() == false))//add validation for Ginger runner tags
+                if (activity.Active != false)
```

Comment:
we also have this check in Business flow, we agreed to remove it  and keep only 1 right ?